docs/en/getting_started/install.md

```diff
@@ -59,6 +59,36 @@ sudo yum install clickhouse-server clickhouse-client
 
 You can also download and install packages manually from here: <https://repo.yandex.ru/clickhouse/rpm/stable/x86_64>.
 
+### From tgz archives {#from-tgz-archives}
+
+It is recommended to use official pre-compiled `tgz` archives for all Linux distributions, where installation of `deb` or `rpm` packages is not possible.
+
+Required version can be dowloaded with `curl` or `wget` from repository <https://repo.yandex.ru/clickhouse/tgz/>.
+After that downloaded archives should be unpacked and installed with installation scripts. Example for the latest version:
+```bash
+export LATEST_VERSION=`curl https://api.github.com/repos/ClickHouse/ClickHouse/tags 2>/dev/null | grep -Eo '[0-9]+\.[0-9]+\.[0-9]+\.[0-9]+' | head -n 1`
+curl -O https://repo.yandex.ru/clickhouse/tgz/clickhouse-common-static-$LATEST_VERSION.tgz
```

Comment:
BTW `curl` is usually less available than `wget`.

Reply:
```
$ docker run --network=host -it archlinux bash
[root@click /]# wget
bash: wget: command not found
[root@click /]# curl
curl: try 'curl --help' for more information

$ docker run --network=host -it gentoo/stage3-amd64 bash
click / # wget
wget: missing URL
Usage: wget [OPTION]... [URL]...

Try `wget --help' for more options.
click / # curl
curl: try 'curl --help' or 'curl --manual' for more information

$ docker run --network=host -it vbatts/slackware bash
bash-4.3# wget
wget: missing URL
Usage: wget [OPTION]... [URL]...

Try `wget --help' for more options.
bash-4.3# curl
bash: curl: command not found
```
parity!